src/full/Agda/Interaction/Highlighting/HTML.hs

```diff
@@ -195,19 +209,29 @@ h !! as = h ! mconcat as
 -- | Constructs the web page, including headers.
 
 page :: FilePath              -- ^ URL to the CSS file.
+     -> Bool                  -- ^ Highlight occurrences
      -> Bool                  -- ^ Whether to reserve literate
      -> C.TopLevelModuleName  -- ^ Module to be highlighted.
      -> Html
      -> Text
-page css htmlHighlight modName pageContent =
+page css
+     highlightOccurrences
+     htmlHighlight
+     modName
+     pageContent =
   renderHtml $ if htmlHighlight
                then pageContent
                else docTypeHtml $ hdr <> rest
   where
 
-    hdr = Html5.head $ mconcat
+    hdr = if highlightOccurrences then hdr' <> highlightHover else hdr'
+    highlightHover = script mempty !!
+      [ type_ "text/javascript"
+      , src $ stringValue occurrenceHighlightJsFile
+      ]
+    hdr' = Html5.head $ mconcat
       [ meta !! [ charset "utf-8" ]
-      , Html5.title (toHtml $ render $ pretty modName)
+      , Html5.title (toHtml . render $ pretty modName)
       , link !! [ rel "stylesheet"
                 , href $ stringValue css
                 ]
```

Comment:
It seems the "script" tag ends up outside of the `<head>...</head>` tags.
Add something like:

```
, if highlightOccurrences then highlightHover else mempty
]
```
so that "<script>...</script>" is inside the head section.